### DISCUSSION TITLE
Default annotations no longer "rich" (for me)

### DISCUSSION THREAD

**Discussion by shipmints** [minad/marginalia#183] [Q&A]
I confess that I blindly updated ```marginalia``` a few times in the last couple of months. Along the way, the default annotations became "plain" rather than "rich" and I have to cycle to reset to rich annotations. This could be user error, for sure, though I haven't changed my config in a while. I'd happily accept a pointer to look in the right place or perhaps other users are experiencing the same thing?

**Comment by minad** [comment]
> This could be user error, for sure, though I haven't changed my config in a while.

Most likely, yes. Marginalia shows rich annotations by default. So please try to narrow down the issue, ideally starting from emacs -Q and keep us posted. Thanks!

**Reply by shipmints** [reply]
Still no idea on root cause, though I did dig around a bit. I implemented the marginalia-cycle advice to leverage savehist and that's working. I dislike mysteries, though.

**Reply by shipmints** [reply]
It actually looks like this, not advice since it need be done only once per Emacs session:
```elisp
(use-package marginalia
...
  (with-eval-after-load 'savehist
    (add-to-list 'savehist-additional-variables 'marginalia-annotator-registry))
...
```
